src/ripple/app/ledger/impl/LedgerConsensusImp.h

```diff
@@ -367,6 +367,24 @@ make_LedgerConsensus (
     LedgerMaster& ledgerMaster,
     FeeVote& feeVote);
 
+//------------------------------------------------------------------------------
+/** Apply a set of transactions to a ledger
+
+  Typically the txFilter is used to reject transactions
+  that already got in the prior ledger
+
+  @param set                   The set of transactions to apply
+  @param view                  ledger to apply transaction to
+  @param txFilter              rule to reject unwanted txns
```

Comment:
It's not really clear from this comment how `txFilter` rejects unwanted txns. Suggest something like `callback returns false to reject unwanted txns`.